pkgs/applications/backup/timeshift/default.nix

```
@@ -0,0 +1,34 @@
+{ callPackage
+, lib
+, timeshift-wrapper ? (callPackage ./wrapper.nix { })
+, rsync
+, coreutils
+, mount
+, umount
+, psmisc
+, cron
+, enableBtrfs ? true, btrfs-progs ? null
+, enableGrub ? true , grubPackage ? grub2_full, grub2_full ? null
```

```suggestion
, enableBtrfs ? true, btrfs-progs
, enableGrub ? true , grubPackage
```
Please default grubPackage to grub2_full in top-level.